### DISCUSSION TITLE
Display a Notification when a new sfdx version is available

### DISCUSSION THREAD

**Discussion by AllanOricil** [forcedotcom/salesforcedx-vscode#5352] [Ideas]
**Is your feature request related to a problem? Please describe.**

Vscode could be a notification channel for new sfdx cli versions.

**Describe the solution you'd like**
Whenever an sfdx project workspace is identified, display a vscode notification asking if the user wants to update his/her cli.

**Describe alternatives you've considered**

Currently, the only way to be notified is when running a sfdx command on a terminal

**Additional context**

I was just going to bed and thought about it. Not sure if this feature exists already.

**Comment by AnanyaJha** [comment]
Hi @AllanOricil , really interesting idea! We don't have support for functionality like this now, but I'll convert this to a Discussion and see what our users think. Thanks for suggesting!
